### ISSUE TITLE
Help Californians remove old felonies from their criminal records

### ISSUE THREAD

**Issue by jennymt** [codeforamerica/project-ideas#64]
Prop 47 (which California voters approved in November 2014) makes it possible for certain low-level, nonviolent felonies to be changed to misdemeanors on old criminal records. To get an old felony removed, people have to fill out a form and file it with the court. In some counties, like LA, they have to also file service of process. Some people represent themselves to file for a record change, but there is also an "assister community" of public defenders, pro bono practices at law firms, and legal services organizations that represent people who are seeking a record change. The project would be to build a tool that makes this process easier and while maintaining high rates of approvals of the record change requests.

The different county forms are available here: http://www.safeandjust.org/recordchange

**Comment by Mr0grog** [comment]
Sounds similar in a lot of ways to http://expunge.io by @cathydeng. Maybe looking at that codebase or talking with her would be a good place to start.

**Comment by greggish** [comment]
cc @justgrimes

**Comment by cathydeng** [comment]
hi!

expunge.io helps people figure out if they're eligible to expunge their juvenile records in Cook County IL, communicates the steps to expunge, & directs them to legal aid

some notes:
- expunge.io doesn't automate applications or file the actual petition to the court - for juvenile records in Chicago, I learned that people often mis-remember the details of their case, or misunderstand legalese, so we thought it was important to always guide someone to an actual human to talk to. also, in Chicago, you need to go in person to the police department to get your RAP sheet, which has the details of all past arrests
- designing the eligibility questions to be simple (language, order, what to keep & what to leave out) was the vast majority of the work
- the geographic spans of the expungement policy (state) vs the police dept someone needs to get arrest records from (town/city police dept) vs the pro bono legal aid (county in our case) vs the court (county in our case) are good to keep in mind, b/c they'll probably vary & can be confusing to communicate

I'd be happy to chat more about expunge.io, the codebase, design decisions, & lessons learned.

**Comment by daguar** [comment]
@cathydeng — Thanks so much!

> for juvenile records in Chicago, I learned that people often mis-remember the details of their case, or misunderstand legalese, so we thought it was important to always guide someone to an actual human to talk to.

This is a _great_ assumption to test with prospective users here. I imagine some have sufficient detail to fill out the form, but talking to the advocates who are doing this manually with clients right now should yield this info.

**Comment by jennymt** [comment]
Yes, Cathy! Thanks for breaking it down. I think your approach sounds
really smart and in line with what I have heard from advocates here in CA
about how they think the process would work best.

On Thursday, February 26, 2015, Dave Guarino notifications@github.com
wrote:

> @cathydeng https://github.com/cathydeng — Thanks so much!
> 
> for juvenile records in Chicago, I learned that people often mis-remember
> the details of their case, or misunderstand legalese, so we thought it was
> important to always guide someone to an actual human to talk to.
> 
> This is a _great_ assumption to test with prospective users here. I
> imagine some have sufficient detail to fill out the form, but talking to
> the advocates who are doing this manually with clients right now should
> yield this info.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/codeforamerica/project-ideas/issues/64#issuecomment-76208026
> .

## 

Jenny Montoya Tansey
Safety and Justice Lead
Code for America http://www.codeforamerica.org/
jenny@codeforamerica.org | 718.757.5635

**Comment by jennymt** [comment]
Another wrinkle that adds urgency to this problem, is that people will only be able to change their records if their request is submitted by 2017, because of how the ballot measure text has been interpreted by the state.

**Comment by fureigh** [comment]
@jennymt, do you have an existing connection to prospective local users? I suspect there will be some users who'll want to (and be able to) fill out the forms themselves and others who'll have more success working with an advocate, but as @daguar said it'd be good to learn more there.

**Comment by spjika** [comment]
Can def connect you to folks in the east bay, through the EBLC and VOA.

Spike
openoakland.org
www.stealingbeautyphotography.com

**Comment by jennymt** [comment]
Agreed @fureigh. I do have some connections to users, both formerly incarcerated, and in the "assister community" of public defenders and legal aid attorneys. I think it would be good to get feedback from both. But I'm wondering if it makes sense to throw together a prototype first? (I'm open to being wrong about that.) The process for reclassification is laid out here: http://myprop47.org/reclassification/

The eligibility criteria are the same across the state, but the paperwork required varies slightly from county to county. 

Challenges include:
there are hundreds of thousands of people eligible, but most don't know about it, they are highly transient, and they need to apply by 2017 to take advantage of the program
limited capacity of legal community to assist everyone who needs help
tough for many people who could benefit to understand "legalese" and the requirements (service of process? where to submit forms?)
If someone or someones wants to work on pulling together a basic prototype, I am happy to work with Spike and my former org, Californians for Safety and Justice, on putting together a session with users. @hackajesse

**Comment by jtashea** [comment]
Hi All. @legalcodes and I put together ExpungeMaryland.org after being expired by Cathy's work in Chicago. One other thing to consider while putting this project together is the limitations around practicing law without a license. If you are considering automating this process without an attorney involved (standards for "involved" will vary state to state) apps like these can get you in trouble.

Happy to talk more if I can be of any help. Good luck!

**Comment by nikzei** [comment]
@jennymt I just received word from a Vallejo public defender that he and other public defenders that are part of a group, Public Defenders for Social Justice, are interested in exploring ways to collaborate with CfA. Let me know if you want me to relay anything to them or to connect you (helpful for user research? user testing?).
Also, I'd love to help with prototyping - let me know.

**Comment by fureigh** [comment]
@nikzei, yesterday I started adapting [Clean](https://github.com/codeforamerica/calfresh-and-so-clean/) to the Alameda County form... sounds like Solano County's form would be a better place to start!

Solano County's form isn't included in the list on http://www.safeandjust.org/recordchange, though. Do you think you can get a copy from that Vallejo public defender?

**Comment by nikzei** [comment]
@fureigh I'll ask for one.

**Comment by jennymt** [comment]
@jtashea i would like to talk more about the unauthorized practice of law issue - does a quick call make sense?

@fureigh @nikzei do you think this needs to be build county by county, or is there a way we could build one tool to rule all the counties?

**Comment by jtashea** [comment]
@jennymt Happy to. You can email me at jtashea@gmail.com and we can set something up. Looking forward to it.

**Comment by fureigh** [comment]
@nikzei Awesome!

@jennymt I imagine we could build one tool to rule all the counties, but because the forms are different from county to county, if the tool is meant to fill out the forms, it'll (eventually, ideally) need to be built to account for all of those cases. Unless there's a legal workaround wherein there's some master form that hasn't yet been mentioned?

**Comment by jennymt** [comment]
@fureigh Unfortunately there is not a common app for changing your record. :( Are you in the office? Maybe we could chat about this for a minute?

**Comment by daguar** [comment]
@fureigh — Let me know if you have any questions re: the Clean adaptation. We've made it muuuch more complex than its early versions, and I think some of the first commits may actually be better for this purpose. The real core of it is `Calfresh::Application#fill_out_form` and a web form that feeds it.

**Comment by nikzei** [comment]
@jennymt @fureigh Since Prop 47 is state law, I would think that one form <i>could</i> serve all counties, and that the current use of different forms is due to lack of coordination? Worth checking with folks in the field? And I think that PDs for Social Justice might be a good resource for this bc it's comprised of people who work in different counties. Happy to field this to them.

**Comment by jennymt** [comment]
Unfortunately (for these purposes), California has a very strong
county-based justice system. Nothing is standardized across counties - from
bail schedules, to what credits you are eligible for, to filing
requirements. I think that the standardization lift would be too heavy,
given that this process will be over the year after next. Some counties
want to make this process easy, others want to make it hard, you know?

On Monday, March 9, 2015, Nikki Zeichner notifications@github.com wrote:

> @jennymt https://github.com/jennymt @fureigh
> https://github.com/fureigh Since Prop 47 is state law, I would think
> that one form _could_ serve all counties, and that the current use of
> different forms is due to lack of coordination? Worth checking with folks
> in the field? And I think that PDs for Social Justice might be a good
> resource for this bc it's comprised of people who work in different
> counties. Happy to field this to them.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/codeforamerica/project-ideas/issues/64#issuecomment-77973596
> .

## 

Jenny Montoya Tansey
Director of Safety and Justice
Code for America http://www.codeforamerica.org/
jenny@codeforamerica.org | 510.909.8317

**Comment by fureigh** [comment]
@jennymt: Thanks for circling back on here. I'm out of the office for the next few days and Slackable, though GitHub Issues has the advantage of being self-documenting. To be clear, lack of standardization isn't a blocker, it just makes scaling more complicated and time-consuming.

@daguar: Good to know, thanks! I'd considered working off of http://github.com/fureigh/medi-cal-fresh-and-so-clean, which is based on a much earlier version of Clean... sounds like that may be a better way to go, yeah? If so, sometime down the road it'd be good to hear your thoughts about which aspects of the newer Clean it makes sense to incorporate.

@nikzei: Maybe couldn't hurt to chat with PDs for Social Justice anyway? Worst case, you raise interest in using a tool that simplifies the process. If you'd rather hold off until there's a prototype to show them, though, I trust your judgment.

**Comment by fureigh** [comment]
@nikzei, I started hacking away at https://github.com/codeforamerica/re-record. Emphasis on the hacking, but something's started to take shape. Feel free to dive in! You're also welcome to run with this in a different direction if you're so inspired.

**Comment by jennymt** [comment]
@nikzei i agree with @fureigh - it would be great to touch base with PDs for Social Justice - they would probably be excited to help out and have some useful info for us! My earlier response was too downer just based on my understanding of the cat-herding qualities of trying to get CA counties on the same page. On another notes, the SF brigade is very interested in helping out on this. I tried tagging Jesse here in github, but he got at me over email. Maybe their many hands could make the scaling easier?

**Comment by spjika** [comment]
when something get's close to workable and a clear path emerges, def
consider swinging over to Oaktown too, would be good to do some paired dev
in two counties!

Spike
openoakland.org
www.stealingbeautyphotography.com

**Comment by jennymt** [comment]
@spjika we will def swing over to Oaktown! My former org, Californians for Safety and Justice, created Prop 47 and is working hard on its implementation. They are based in Oakland and are really excited about this effort. I was just talking with them about your offer to round up some users over there - twill be a great event!

**Comment by nikzei** [comment]
@fureigh sounds good! @jennymt no worries - sounds frustrating! Though since Prop 47 provides such a clear statewide standard, I'm not sure why there should be any meaningful differences county by county in at least the information that they would need to gather for an application. I'd love to talk about it more - would be worthwhile to understand better how counties are processing applications.

**Comment by jennymt** [comment]
I think the differences mainly exist for two reasons: 1) differences between information systems and the information each county feels they need to find someone's record and all their potentially eligible offenses; 2) as I said before, differences in how easy or hard counties want to make the process. The DA's association was vehemently opposed to this measure, and they definitely had a hand in crafting the forms at the county level. See, for example, Fresno's form: http://www.fresno.courts.ca.gov/criminal/PCR-80%20Application%20for%20Reduction%20of%20Felony%20Conviction%20-%20Proposition%2047.pdf

Fresno asks for your CI&I number (honestly, I don't even know what that is); your CDCR number (state prison number - which you might not even have if your conviction was from 2011 or later); and in addition to the case number and date for your convictions they also want to know the term imposed. None of that is required by state law, but there it is on the form.

**Comment by spjika** [comment]
I love that question about "are you requesting a hearing?" - given most
people won't really read the procedure document, this is a terrible
question- you either think "oh crap I have to do this to get it done" or
you think that not doing it must cancel you application process. Bad
placement and instruction.

I'd be curious if people had confusion over their CDCR No versus their Life
No- CDCR uses both, only a person can get multiple CDCR numbers over their
life, but only ever have one LifeNo. Have to ask some folks what they
understand they have and know.

CI&I is just a local courts number/ID.

Spike
openoakland.org
www.stealingbeautyphotography.com

**Comment by fureigh** [comment]
Do we know anyone who could find out whether, as was the case with CalFresh, it's possible to submit a form with only the legally required fields without jeopardizing the case's odds? If so, that'd dramatically simplify the experience for everyone.

If not, once this is refactored a bit and there's a HOWTO on adding your county's form, many hands will absolutely be helpful in scaling.

Something to note there: some of the counties use Microsoft Word documents instead of PDFs. Not a big deal, we can just make PDF forms of the Word docs, but it's another piece to handle.

@nikzei, looking forward to hearing what you learn about the Solano form. We could pair on incorporating it/documenting the process of adding a county, as the current draft uses Alameda County's form. Or if you want to run with it (at a Brigade or otherwise) in the meantime, go for it.

**Comment by spjika** [comment]
Anyone asked the question of will any counties accept something not on
their form? The Alameda County form has half the spaces for official use
and for assessing the case, I really have no idea if they'd take a set of
fields not on their real form- is the Clean app going to send them data
packets or a recreated form of their own design? Forgive ignorance here...

Spike
openoakland.org
www.stealingbeautyphotography.com

**Comment by jennymt** [comment]
The repository of forms page at http://www.safeandjust.org/recordchange says that if your county has not created a form, you can use this sample form:
http://libcloud.s3.amazonaws.com/211/f2/8/516/P47_Form_111014v1.pdf

We could make a few calls and ask the courts in some counties if they will accept petitions that are not on their county form. @nikzei maybe spike's question is a good one for PDs for Social Justice - what do PDs think about how courts would receive petitions not on their county's form? I tend to err towards thinking we should be filling out the forms provided, but its worth looking into. 

I suspect Solano county has not made their own special form, which is why it's not linked to. Be curious to see what they say!

Looking over some of these forms, I also worry some about making this process easy for the DIY applicant. To make the tool helpful, we will have to do a lot of translation from county form legalese that would perhaps better be done by a local PD. Maybe the tool should be more oriented towards providing basic information about eligibility and the process and then directing the applicant towards local attorneys - a la expunge.io 

my 102 cents

**Comment by daguar** [comment]
I'd strongly suggest writing to the county-specific forms for now — making it work in one county is a good prerequisite for others anyway, and a ton of learning will occur in that first county context.

**Comment by evanwolf** [comment]
Is there a component to expungement that benefits from community support? Could social elements matter? Or is this mostly a private process?

**Comment by fureigh** [comment]
+1 to what @daguar said. When I referenced Clean above, it was shorthand for "show the user a friendly web form and use their input to fill out the original, potentially more confusing PDF on the back end."

In other words: if we do proceed with a webform the user can fill out, the set of fields will be determined by what county they select. And the PDF filled out behind the scenes will be the corresponding legal form for that county.

**Comment by fureigh** [comment]
Re: making this process easy for the DIY applicant: the forms are already available online, no? I continue to suspect there will be some users who'll want to (and be able to) fill out the forms themselves and others who'll have more success working with an advocate or legal representation.

I think this and the many other excellent questions above, plus where the pain points currently are and what kind of tool, if any, would be most helpful, will be great to discuss with prospective local users.

**Comment by fureigh** [comment]
Screenshot of [Alameda County prototype-in-progress](http://www.github.com/codeforamerica/re-record): ![change your record](https://cloud.githubusercontent.com/assets/1244599/6592401/eb040370-c78a-11e4-975f-7afbd40eb26a.png)

Lots of [room for improvement](http://www.github.com/codeforamerica/re-record/issues), but hopefully this is a little clearer. Wordy intro text temporarily borrowed from CSJ.

**Comment by jennymt** [comment]
Yippee!!!! A prototype in progress! 💜💜💜

On Wednesday, March 11, 2015, Fureigh notifications@github.com wrote:

> Screenshot of Alameda County prototype-in-progress
> http://www.github.com/codeforamerica/re-record: [image: change your
> record]
> https://cloud.githubusercontent.com/assets/1244599/6592401/eb040370-c78a-11e4-975f-7afbd40eb26a.png
> 
> Lots of room for improvement
> http://www.github.com/codeforamerica/re-record/issues, but hopefully
> this is a little clearer. Wordy intro text temporarily borrowed from CSJ.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/codeforamerica/project-ideas/issues/64#issuecomment-78220614
> .

## 

Jenny Montoya Tansey
Director of Safety and Justice
Code for America http://www.codeforamerica.org/
jenny@codeforamerica.org | 510.909.8317

**Comment by jennymt** [comment]
@fureigh I am so delighted that this prototype in progress exists! It seems like we're close to being in a place to show it to some users in Alameda County, which is exciting. some questions: 1) I am assuming because you adapted clean that this form populates the actual county form? 2) @daguar tells me that some iteration of clean provides a copy to print for your files. we definitely want that feature in this tool - do you know if its in here? Legal assisters and self-filers will both def want to keep a copy. 2) my understanding of filing in alameda county is that you must snail mail the form to the courthouse or drop it off in person, as well as sending a copy of the form to the DA's office - as opposed to emailing a PDF, as clean does for cal-fresh apps. can we adapt so that the tool mails a copy to the court and DA? Alternatively, we could just have people print and mail themselves, and get feedback from users about both options. I'm sure there's some cost for the option where the tool mails it, but I'm suspect either CfA or CSJ could underwrite that. Eeeeeeeee!!

**Comment by nikzei** [comment]
@fureigh so cool! @jennymt my pd contact in Vallejo says that his office uses the San Diego form, which is not listed on the safeandjust.org page nor is it on the San Diego public defender's site nor the San Diego County Court website. 

I would suggest calling a few county courts to see what types of forms they'd accept; I would imagine that there's flexibility re: forms, particularly given that particular forms are not always provided by the courts themselves. But if you call some courts up and ask, the clerks'll tell you what their requirements are.

**Comment by fureigh** [comment]
@nikzei, good to know, thanks! Do you want to pursue getting a copy of San Diego's form somehow? (ccing @MrMaksimize in case he has a suggestion on that front)

@jennymt, do you have bandwidth to call courts and ask about requirements/propose solutions?

To your questions:
1. It's designed to populate a PDF of the actual [Alameda County form](http://www.alameda.courts.ca.gov/Resources/Documents/01%20Local%20Form%20CRM-050%20Petition%20for%20Resentencing%20v3%2001%281%29.pdf), yes. We'll want confirmation that all necessary fields (but no more!) are included and that the labels and placeholder text are accurate (https://github.com/codeforamerica/re-record/issues/3).
2. I don't think that's currently included, but it could be incorporated.
3. Cost is likely the limiting factor here. On the technical side, I'd look at http://trypaper.com. Perhaps a nonprofit/bulk discount or a donation could be negotiated? The non-discounted rate starts at about a dollar per mailing.

For more context, [here's the current list of issues](https://github.com/codeforamerica/re-record/issues).

**Comment by fureigh** [comment]
http://myprop47.org has a lot of useful info. This may be putting the cart before the horse, but I think this could eventually be useful to that site, if Californians for Safety and Justice is into it. (Why, hello again, Californians for Safety and Justice!)

The current prototype could help with steps 2 through 4 of [this infographic](http://myprop47.org/app/uploads/2015/02/4.-Reclassification_Infographic.pdf). I'd love to get more user feedback, including validation/invalidation of that idea, before digging in too much more. @jennymt, maybe we can chat tomorrow or Wednesday about how best to proceed?

**Comment by jennymt** [comment]
Before I even created this issue in github, I made a little texting tool (with clutch guidance from @daguar and the health lab) that does an eligibilty screen for a Prop 47 record change. I used a random number I got from twilio for it. (If you want to check it out: Text (818) 473-2865 to find out if you are eligible to reclassify your felony under prop 47. ) 

Now I want to get it a good number like PROP47 or MYPROP47 or something. Does anyone know how I can buy a specific, possibly less than 10 digit, number like that?

**Comment by jennymt** [comment]
And then I immediately found what I was looking for: http://www.usshortcodes.com/

**Comment by daguar** [comment]
Just a heads up that short codes are pretty expensive I think: check them out on Twilio as a first stop.

**Comment by jennymt** [comment]
Yeah I think they run $1000/month - or so wikipedia tells me.

**Comment by daguar** [comment]
Yup, BUUUUT in Twilio you can try to create a long number using letters. Try it out in their "buy a phone #" interface.

**Comment by jennymt** [comment]
Oh, believe me I have tried that. Have yet to come up with anything workable. But you may have inspired me to give it another go.

**Comment by fureigh** [comment]
Google Voice might be another route to consider.

**Comment by DarkJex** [comment]
ĤĨŔĔ Ă РŔŐŦĔŚŚĨŐŃĂĹ ĂŃĎ ĞĔŤ ŤĤĔ βĔŚŤ ŔĔŚÚĹŤŚ ĨŃ Ă ŚĤŐŔŤ ĞĨVĔŃ ŤĨМĔ.


ĎŐ ŶŐÚ ŔĔQÚĨŔĔ ĂŃŶ ŐŦ ŤĤĔ ŦŐĹĹŐŴĨŃĞ ŚĔŔVĨČĔŚ ?


*ĤĂČĶ ĔМĂĨĹŚ ĂŃĎ ŚŐČĨĂĹ МĔĎĨĂ ĂČČŐÚŃŤŚ

*ĤĂČĶ МŐβĨĹĔ РĤŐŃĔŚ ĂŃĎ ŚĔŔVĨČĔ РŔŐVĨĎĔŔŚ

*ĤĂČĶ ÚŃĨVĔŔŚĨŤŶ ĞŔĂĎĔ ĂŃĎ ŤŔĂŃŚČŔĨРŤ

*ČĹĔĂŔ ČŔĨМĨŃĂĹ ŔĔČŐŔĎŚ ĂŃĎ ĎМV ŔĔČŐŔĎŚ

*ČĹĔĂŔ ĎĔβŤ ĂŃĎ МŐŔŤĞĂĞĔ

*ŚĔČÚŔĨŤŶ βŔĔĂČĤĔŚ ĂŃĎ ŔĔVĔŃĞĔ ĤĂČĶ

*ŚĔĂŔČĤ ĔŃĞĨŃĔ ŐРŤĨМĨŹĂŤĨŐŃŚ

*ĹŐĂĎĔĎ ČŔĔĎĨŤ ČĂŔĎŚ ĂŃĎ ŴĔŚŤĔŔŃ ÚŃĨŐŃ ĤĂČĶŚ

*ŔĔČŐVĔŔĨŃĞ ŐŦ βĨŤČŐĨŃ ŴĂĹĹĔŤ ŐŔ ŚŤŐĹĔŃ βĨŤČŐĨŃ

*βĨŤČŐĨŃ МĨŃĨŃĞ


ĔМĂĨĹ: ĤĂČĶĔŔŚĎŐММ@ĞМĂĨĹ.ČŐМ

ŤĔĹĔĞŔĂМ: ĤŤŤРŚ://Ť.МĔ/ĤĂČĶĔŔŚĎŐММ
